src/topology/constructions.lean

```diff
@@ -765,6 +765,16 @@ begin
       simpa [pi_if, hf] } }
 end
 
+lemma inducing_infi_to_pi {X : Type*} [∀ i, topological_space (π i)] (f : Π i, X → π i) :
+  @inducing X (Π i, π i) (⨅ i, induced (f i) infer_instance) _ (λ x i, f i x) :=
```

Comment:
Could we have a doc-string on this one? It took me a while to work out what it was even saying.

Reply:
Good idea! Done.